festim/exports/exports.py

```diff
@@ -133,6 +133,6 @@ def initialise_derived_quantities(self, dx, ds, materials):
         """
         for export in self:
             if isinstance(export, festim.DerivedQuantities):
-                export.data = [export.make_header()]
+                export.data = []
```

Comment:
Should we do the same for `export.t` 
```suggestion
                export.data = []
                export.t = []
```